packages/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -1540,9 +1541,12 @@ export class PhysicalModelConsolidator extends IModelTransformer {
   public constructor(
     sourceDb: IModelDb,
     targetDb: IModelDb,
-    targetModelId: Id64String
+    targetModelId: Id64String,
+    argsForProcessChanges?: ProcessChangesOptions
   ) {
-    super(sourceDb, targetDb);
+    super(sourceDb, targetDb, {
+      argsForProcessChanges,
```

Comment:
Before committing and pushing these changes, Prettier runs and formats the files. Adding the comma at the end is part of its formatting rules, so if it is removed, it will be added back